### ISSUE TITLE
ID3 album art displays incorrectly

### ISSUE THREAD

**Issue by stkAnon** [gnumdk/lollypop#311]
Most of these albums have their own id3 art embedded in their mp3 files, but they all show the same image.

![screenshot from 2015-10-12 15-37-53](https://cloud.githubusercontent.com/assets/8258259/10440864/acc4e568-70f7-11e5-8712-74b56381d168.png)
![screenshot from 2015-10-12 15-38-16](https://cloud.githubusercontent.com/assets/8258259/10440865/ae66ddd6-70f7-11e5-91d5-6980f86e5991.png)

If it's relevant, I managed to get it to briefly display properly by disabling 
my network connection and pressing "Update music".

**Comment by gnumdk** [comment]
They show the same image because you have a jpg files in mp3 folder.

Lollypop search for:
- cover.jpg
- any jpg file
- embedded art
- lastfm
- itunes
- spotify

**Comment by stkAnon** [comment]
I think Lollypop's creating the .jpg files every time it updates the library...  I definitely didn't put them there myself. Strangely enough, it always saves the exact same two images every time (the ones I have in the second screenshot), which it then uses as album art for all the other songs (despite them having their own embedded id3 covers).

**Comment by stkAnon** [comment]
I just took a quick look at the two albums that the .jpg's belong to, and neither of them have embedded cover art in their mp3's. So that's probably why Lollypop is downloading and saving those two images specifically. 

I personally find it a bit strange that those files are being saved in the same place as my .mp3 library and that they override embedded album art from other albums though (I assume this second one isn't intended behaviour?)

Also, I just noticed some new behaviour the last time I started the program. The majority of albums are still displayed with the same image, but some seemingly random ones are shown with their embedded album art.
![screenshot from 2015-10-13 16-29-23](https://cloud.githubusercontent.com/assets/8258259/10471221/b9d032e0-71c8-11e5-8704-0def6a39e141.png)
